pkg/driver/nri_hooks.go

```diff
@@ -232,7 +230,7 @@ func (np *NetworkDriver) StopPodSandbox(ctx context.Context, pod *api.PodSandbox
 	klog.V(2).Infof("StopPodSandbox Pod %s/%s UID %s", pod.Namespace, pod.Name, pod.Uid)
 	start := time.Now()
 	defer func() {
-		np.netdb.RemovePodNetns(podKey(pod))
+		np.netdb.RemovePodNetNs(podKey(pod))
```

Comment:
nit: isn't `netns` typically written as one word in linux? (so the previous form was correct?)

Reply:
Given containerd and NRI have NetNS primarily, let's lave it.